src/devices/RpiCpuTemperature/RpiCpuTemperature.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.IO;
+using System.Runtime.InteropServices;
+
+namespace Iot.Device.Rpi
+{
+    public class CpuTemperature
+    {
+        public double? ReadTemperature()

```

Comment:
please use property and change to private
```C#
public double? Temperature => ReadTemperature();
```

Reply:
Or use static methods?

Reply:
I'd just go with double and throw elsewhere and perhaps have another static property `IsAvailable` or something along these lines